### DISCUSSION TITLE
Previously stored elements/nodes do not match the current element/nodes

### DISCUSSION THREAD

**Discussion by jin0123456** [idaholab/moose#27759] [Q&A Meshing]
Hello,

I'm trying to recover a simulation when this error ocurred,
`RestartableEquationSystems::load(): Previously stored elements/nodes do not match the current element/nodes`
But what I reloaded is the mesh from the last step,
```
[Mesh]
  [simple_mesh]
    type = FileMeshGenerator
    file = Rod-15309_thm_checkpoint_cp/0034-mesh.cpr
  []
[]
```
So what  does it represent?

**Comment by GiudGiud** [comment]
Hello

How are you setting up the recovery?
From the problem (as a restart) or from the command line?

Guillaume

**Reply by jin0123456** [reply]
Sorry, I restarted the simulation by `Problem`.

**Reply by GiudGiud** [reply]
What does your problem block look like ?

Can you try a recover from the command line?

**Reply by jin0123456** [reply]
Problem block,
```
[Problem]
  restart_file_base = Rod-15309_thm_checkpoint_cp/LATEST
  force_restart = true
  coord_type = RZ
[]
```
My first simulation ran really well, and I need to change some conditions based on results from the last step of the first simulation.
So I suppose I need restart?

**Reply by GiudGiud** [reply]
force_restart is only for restart from multiapps. I recommend you remove this unless this is in a multiapp

**Reply by jin0123456** [reply]
OK.
